caffe2/python/operator_test/_utils.py

```diff
@@ -0,0 +1,47 @@
+import torch
+from typing import Tuple, Any, Optional
+
+_DTYPE_PRECISIONS = {
+    torch.float16: (1e-3, 1e-3),
+    torch.float32: (1e-4, 1e-5),
+    torch.float64: (1e-5, 1e-8),
+}
+
+
+def _get_default_rtol_and_atol(actual: torch.Tensor, expected: torch.Tensor) -> Tuple[float, float]:
+    actual_rtol, actual_atol = _DTYPE_PRECISIONS.get(actual.dtype, (0.0, 0.0))
+    expected_rtol, expected_atol = _DTYPE_PRECISIONS.get(expected.dtype, (0.0, 0.0))
+    return max(actual_rtol, expected_rtol), max(actual_atol, expected_atol)
+
+
+# The function below is a faithful replica of the former `torch.testing.assert_allclose`. This is only here,
+# because it is used extensively throughout the tests in this package while needing one feature that
+# the new `torch.testing.assert_close` does not offer: comparison between numpy arrays and torch tensors. See
+# https://github.com/pytorch/pytorch/issues/61844 for the reasoning why this feature was removed.
```

Comment:
Per comment. This only applies to the tests located in `caffe2/python/operator_test/*`.